src/index.ts

```diff
@@ -76,10 +78,25 @@ const maxLevelWrite: pino.WriteFn = function(this: any, data: object): void {
 };
 
 const initFormatters = (options: CosmasOptions & { loggerName?: string }) => {
+    const pkgJson = JSON.parse(fs.readFileSync(path.resolve(path.join(__dirname, '..', 'package.json')), 'utf8'));
+
     const formatters: pino.LoggerOptions['formatters'] = {};
-    if (options.pretty || options.disableStackdriverFormat) return formatters;
-    formatters.level = (_label: string, level: number) => {
-        return { level, severity: PINO_TO_STACKDRIVER[level] || 'UNKNOWN' };
+    if (!options.pretty && !options.disableStackdriverFormat) {
+        formatters.level = (_label: string, level: number) => {
+            return { level, severity: PINO_TO_STACKDRIVER[level] || 'UNKNOWN' };
+        };
+    }
+
+    // do not put logger name field to pretty outputs
+    formatters.log = (object: { [key: string]: any }) => {
```

Comment:
This block would be way readable imo, if the same conditions were condensed into nested if or if a single top-level if was used:

```
if (!options.pretty) {
   formatter.log = () ... {
     object[pkgVersionKey] = pkgJson.version;
     if (options.loggerName) { object[loggerNameKey] = options.loggerName; }
  }
}
```

Reply:
Changed in 6dce7fac29447ec52f0ea9582adbca4b8a26e9d8